config/cloudkeeper.yml

```diff
@@ -24,4 +24,5 @@ cloudkeeper:
   logging:
     level: ERROR # Logging level
     file: /var/log/cloudkeeper/cloudkeeper.log # File to write log to. To turn off file logging leave this field empty.
+  lock-file: /var/lock/cloudkeeper # File used to ensure only one running instance of cloudkeeper
```

Comment:
That works only because `/etc/init.d/cloudkeeper-cron` is only ever executed by `root`. Here, it would be a problem. Thanks!